tasks/install.yml

```diff
@@ -2,39 +2,39 @@
 
 # The standard ca-certs are needed because  without them apt_key will fail to
 # validate www.postgresql.org (or probably any other source).
-- name: PostgreSQL | Make sure the CA certificates are available
+- name: PostgreSQL | Make sure the CA certificates are available | apt
   apt:
```

Comment:
Things weren't tagged with apt/yum... so I've added this.  It's a bit out of the scope of this change, but since I was changing `install_yum.yml` anyway, I figured I would make it more consistent.